src/ontology/edits/oeo-shared.omn

```diff
@@ -1535,6 +1535,9 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/724"
         <http://purl.obolibrary.org/obo/IAO_0000030>
     
     
+Class: OEO_00000139
```

Comment:
Why this line?

Reply:
Seems `electrical energy` (OEO_00000139) needs to be moved to oeo-shared.

Reply:
Obviously not fully. It is only this line that is called.

Reply:
Did you intend to move it?

Reply:
I just moved the class, else OEO_00000139 would be a runaway class in oeo-shared. For the same reason, I also moved `service` (OEO_00140124).